domains/transfer/Transfer.API/API/Transfer/Api/Exceptions/BusinessException.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Net;
+
+namespace API.Transfer.Api.Exceptions;
+
+public class BusinessException : Exception
```

Comment:
I am abit unsure if this is breaking something. Can see today we use: ValidationProblem with 409 conflict many places. Just so we don't suddenly change existing status codes.